sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldText.cs

```diff
@@ -22,12 +24,16 @@ internal FieldText(string text, int pageNumber, BoundingBox boundingBox, IReadOn
         }
 
         /// <summary>
+        /// A list of refereces to the text elements constituting this <see cref="FieldText"/>.
         /// </summary>
         public IReadOnlyList<FormContent> TextContent { get; }
```

Comment:
Wanted to double check first. This is only populated when `IncludeTextContent` is set to `true`, correct? If that's the case, I'll update the description to match other `TextContent` properties.

Reply:
Yes, that's correct.  Thanks!